src/RepoDriver.sol

```diff
@@ -0,0 +1,508 @@
+        bytes memory payload = _requestPayload(forge, name);
+        bytes memory callData = abi.encodeWithSelector(
+            OperatorInterface.operatorRequest.selector,
+            address(0), // ignored, will be replaced in the operator with this contract's address
+            0, // ignored, will be replaced in the operator with the fee
+            storageRef.jobId,
+            this.updateRepoOwnerByAnyApi.selector,
+            nonce,
+            2, // data version
+            payload
+        );
+        require(linkToken.transferAndCall(operator, fee, callData), "Transfer and call failed");
+        // slither-disable-next-line reentrancy-events
+        emit RepoOwnerUpdateRequested(repoId, forge, name);
+    }
+
+    /// @notice Builds the AnyApi generic `bytes` fetching request payload.
+    /// It instructs the operator to fetch the current owner of the repository.
+    /// @param forge The forge where the repository is stored.
+    /// @param name The name of the repository.
+    /// @return payload The AnyApi request payload.
+    function _requestPayload(Forge forge, bytes memory name)
+        internal
+        view
+        returns (bytes memory payload)
+    {
+        // slither-disable-next-line uninitialized-local
+        BufferChainlink.buffer memory buffer;
+        buffer = BufferChainlink.init(buffer, 256);
+        buffer.encodeString("get");
+        buffer.encodeString(_requestUrl(forge, name));
+        buffer.encodeString("path");
+        buffer.encodeString(_requestPath());
+        return buffer.buf;
+    }
+
+    /// @notice Builds the URL for fetch the `FUNDING.json` file for the given repository.
+    /// @param forge The forge where the repository is stored.
+    /// @param name The name of the repository.
+    /// @return url The built URL.
+    function _requestUrl(Forge forge, bytes memory name)
+        internal
+        pure
+        returns (string memory url)
+    {
+        if (forge == Forge.GitHub) {
+            return string.concat(
+                "https://raw.githubusercontent.com/", string(name), "/HEAD/FUNDING.json"
+            );
+        } else if (forge == Forge.GitLab) {
+            return string.concat("https://gitlab.com/", string(name), "/-/raw/HEAD/FUNDING.json");
+        } else {
+            revert("Unsupported forge");
+        }
+    }
+
+    /// @notice Builds the JSON path inside `FUNDING.json` where the repository owner is stored.
+    /// @return path The comma-separated JSON path.
+    function _requestPath() internal view returns (string memory path) {
+        // slither-disable-next-line uninitialized-local
+        string memory chainName;
+        if (block.chainid == CHAIN_ID_ETHEREUM) chainName = CHAIN_NAME_ETHEREUM;
+        else if (block.chainid == CHAIN_ID_GOERLI) chainName = CHAIN_NAME_GOERLI;
+        else revert("Unsupported chain");
+        return string.concat("drips,", chainName, ",ownedBy");
```

Comment:
I wouldn't call it `ownedBy`. Technical this is correct, that the `repoId` is controlled by this address.
The github repo on the other hand is not 'owned' by this address.
However, from the project perspective it is more `projectAddress` or something.

Reply:
You're right, it could be understood as the GitHub or GitLab repo ownership. So maybe `controlledBy`? OTOH it lives under `drips->ethereum->ownedBy`, so it's clear that it's the ownership in the context of Drips. `projectAddress` seems confusing, many users will just paste their wallet address there or the team's mulstisig, which is not a project.

Reply:
Yeah, no strong opinion.

Reply:
I'd stick with `ownedBy`, it's a common concept in crypto and `controlledBy` doesn't clear any confusion about the context.